packages/validator/src/services/utils.ts

```diff
@@ -59,3 +80,38 @@ export function groupSyncDutiesBySubCommitteeIndex(
 
   return dutiesBySubCommitteeIndex;
 }
+
+/**
+ * Return signature in bytes. Assumption that the pubkey has it's corresponding secret key in the keystore of the remote signer.
+ */
+export async function requestSignature(
+  pubkey: string | BLSPubkey,
+  signingRoot: string | Uint8Array,
+  endpoint: string
+): Promise<Uint8Array> {
+  await init("blst-native");
+  const pubkeyHex = typeof pubkey === "string" ? pubkey : toHexString(pubkey);
+  const signingRootHex = typeof signingRoot === "string" ? signingRoot : toHexString(signingRoot);
+  const body = {
+    signingRoot: signingRootHex,
+  };
+
+  try {
+    const url = `${endpoint}/sign/${pubkeyHex}`;
+    const headers = {
+      "Content-Type": "application/json",
+    };
+
+    const res = await fetch(url, {
```

Comment:
@dapplion should we implement the whole API and store it in `lodestar-api`?

Reply:
I don't think so, it's a different set of APIs part of different standards